mmcv/ops/feature_refine.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext(
+    '_ext', ['feature_refine_forward', 'feature_refine_backward'])
+
+
+class FeatureRefineFunction(Function):
+    """Feature refine class."""
```

Comment:
Please add the docstring about what did the ops do and information about input and output.